tensorflow_recommenders/tasks/retrieval.py

```diff
@@ -109,10 +114,10 @@ def call(self,
       candidate_embeddings: [num_queries, embedding_dim] tensor of candidate
         representations.
       sample_weight: [num_queries] tensor of sample weights.
-      candidate_sampling_probability: Optional tensor of candidate sampling
+      candidate_sampling_probability: Optional [m, n] tensor of candidate sampling
```

Comment:
Could you spell out what `m` and `n` are here?

Reply:
oops, yep.